cluster/gce/gci/configure.sh

```diff
@@ -293,10 +293,10 @@ EOF
   fi
 
   echo "Downloading crictl"
-  local -r crictl_path="https://storage.googleapis.com/kubernetes-release/crictl"
+  local -r crictl_path="https://storage.googleapis.com/k8s-artifacts-cri-tools/release/${crictl_version}"
   download-or-bust "${crictl_sha1}" "${crictl_path}/${crictl}"
-  mv "${KUBE_HOME}/${crictl}" "${KUBE_BIN}/crictl"
-  chmod a+x "${KUBE_BIN}/crictl"
+  tar xf "${crictl}"
```

Comment:
A little late but if we are going to unpack the whole tar file it would be good to use -C (preferably to tmp) to make sure we aren't accidentally leaving a file where it doesn't belong.

Reply:
I think we can follow-up on this on the master branch, right? :) We usually only pack the binary in the tarballs in cri-tools.